.travis.yml

```diff
@@ -3,6 +3,10 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+addons:
+  apt:
+    packages:
+    - go-md2man
```

Comment:
I tried to install it for travis tests to pass, but its not available in `precise` repos. Due to that `test_add_help_plugin` fails.

I'll play with faking a binary with a simple script instead